Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM apache/airflow:2.2.5
+
+RUN pip install --no-cache-dir sqlalchemy psycopg-binary
```

Comment:
I think the [official yaml file](https://airflow.apache.org/docs/apache-airflow/2.1.3/start/docker.html) already contain those packages

Reply:
i see. and even if we do need additional Python deps, we can probably make use of the `_PIP_ADDITIONAL_REQUIREMENTS` environment variable for it (or modify its default)

Reply:
yeah, that should work, though I did not try setting that.

Reply:
ye. testing now without additional python deps

Reply:
```python
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/sqlalchemy/engine/base.py", line 1277, in _execute_context
    cursor, statement, parameters, context
  File "/home/airflow/.local/lib/python3.6/site-packages/sqlalchemy/engine/default.py", line 608, in do_execute
    cursor.execute(statement, parameters)
psycopg2.errors.UndefinedColumn: column dag.concurrency does not exist
LINE 1: ..., dag.schedule_interval AS dag_schedule_interval, dag.concur...
```

interesting, i get this issue when using 2.1.3 docker-compose. seems though that sqlalchemy is definitely there (and i'd guess psycopg is there too given airflow internally relies on pg and airflow itself is written in python)

Reply:
let me take a look...

Reply:
nvm. seems that the previous version 2.2.5 had some built-in mechanism to ensure airflow-init is called if not already initialized when running `docker-compose up`; 2.1.3 doesn't have that.

Reply:
version 2.1.3 works well on my side. Did you run this
`docker-compose down --volumes --rmi all`
to clean up version 2.2.5 before running `docker-compose up airflow-init` for version2.1.3? If not that may be causing the problem.

Reply:
yes, cleaning up them and reinit was the way